compiler/rustc_feature/src/unstable.rs

```diff
@@ -552,8 +552,6 @@ declare_features! (
     (unstable, large_assignments, "1.52.0", Some(83518)),
     /// Allow to have type alias types for inter-crate use.
     (incomplete, lazy_type_alias, "1.72.0", Some(112792)),
-    /// Allows `if/while p && let q = r && ...` chains.
-    (unstable, let_chains, "1.37.0", Some(53667)),
```

Comment:
This should be moved to accepted.rs, and the stabilization version should be set to `1.88.0`.